### ISSUE TITLE
Editing in nearby linea

### ISSUE THREAD

**Issue by amiruna** [Floobits/floobits-sublime#220]
Hi,

If me and my collaborator edit the same document in nearby lines (Sublime-> Floobits), it scrambles the text. It seems that whenever I type something, there is a tolerance zone in which he cannot write without messing up the text. Any ideas why and how this could be fixed? Many thanks!

**Comment by ggreer** [comment]
Yes, this is a known issue. It has to do with how we include some context bytes around the actual changed text. We've been putting this off for a while, sorry.

**Comment by saveman71** [comment]
Hi! We're using Floobits with my team with great success so far, thank you very much for the great product and all the supported editors :)

Unfortunately, we also suffer from the issue. Anyway, good to know where it's coming from!
